protos/shared/shared.proto

```diff
@@ -57,7 +57,7 @@ message RuntimeInfoReport {
   ];
   bytes git_hash = 7; //the git hash of the build related the git repository's branch
   bytes version = 8;
-  bytes db_status_extra = 9; //RESERVED: JSON format structure to store db status information
+  repeated bytes db_status_extra = 9; //RESERVED: JSON format structure to store db status information
   bytes rd_status_extra = 10; //RESERVED: JSON format structure to store redis status information
```

Comment:
Since main db can be extra, why can't read only db be `repeated`? Probably need to think about pairing db and read only db as well.

Reply:
are you saying to change ```bytes rd_status_extra = 10``` to ```map<string, string> rd_status_extra = 10 ?```

Reply:
Oh, sorry, ok, that makes sense. No need to change it.

Reply:
okay.